jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  restoreMocks: true,
+  setupFiles: ['./test/setup.js', './test/env.js'],
+  testMatch: ['**/ui/app/pages/swaps/**/?(*.)+(test).js'],
+};
```

Comment:
Can we please add coverageThreshold for Jest? Something like we have in the Controllers repo: https://github.com/MetaMask/controllers/blob/develop/package.json#L108
We can start with whatever numbers we get now from the coverage report and slowly but surely keep increasing them with new code.

Once we have this minimum threshold set up, I can send a PR for setting up enforcement of that threshold on every Git push. It could be done e.g. via the `husky` node module: https://github.com/typicode/husky
![image](https://user-images.githubusercontent.com/80175477/114109886-14b1ba00-988b-11eb-87b9-598b78cb52dd.png)

Then whoever is contributing to the Swaps feature (and later on to any feature in this repo) will have to add enough unit tests, otherwise they won't be able to push their code.

Let me know what you think.

Reply:
Setting a minimum coverage goal and ratcheting it up is a good idea - that's basically what we've done with the `test:unit:strict` and `test:unit:lax` scripts (the "strict" ones have 100% coverage).

Not sure it needs to go in at this point though? The coverage is incredibly low at the moment. Seems like it could wait for another PR.

The Husky suggestion is bound to be controversial :sweat_smile: Some people find husky very irritating. Certainly a discussion to have elsewhere. I've been considering removing the husky setup from the controllers repo and replacing it with something where the git hook is still version-controlled but is opt-in rather than automatic, so that people who prefer not using it or have their IDE running the linter/tests on each change already don't have to add `--no-verify` to every command.